tests/test_okimat.py

```
@@ -806,21 +809,23 @@ async def test_massage_mode_step(
 class TestOkimatPositionNotifications:
     """Test Okimat position notification handling."""
 
-    async def test_start_notify_no_support(
+    async def test_start_notify_supported(
         self,
         hass: HomeAssistant,
         mock_okimat_config_entry,
         mock_coordinator_connected,
```

Ruff (0.14.13)

Unused method argument: `mock_coordinator_connected`

(ARG002)